kubernetes-maven-plugin/doc/src/main/asciidoc/inc/_introduction.adoc

```diff
@@ -312,23 +312,29 @@ An extract of the plugin configuration is shown below:
       <deployment> <!--11-->
         <my>deployment</my>
       </deployment>
+      <service>
+        <property> <!--12-->
+          <name>cloud.google.com/neg</name>
+          <value>{"ingress":true}</value>
+        </property>
+      </service>
```

Comment:
Do you think [Labels And Annotations section](https://eclipse.dev/jkube/docs/kubernetes-maven-plugin/#labels-annotation-configuration) would be a more appropriate place for this?